src/app/views/cheat-sheets/balancers/balancers.component.html

```diff
@@ -45,12 +45,14 @@
                     <a href="{{sheetData?.commonBalancers[1].url}}" target="_blank" rel="noopener">8<i class="fas fa-long-arrow-alt-right"></i>8</a> throughput unlimited.
                 </li>
                 <li>
-                    Being throughput unlimited input/output amount can vary thus those two cover most cases.
+                    Being throughput unlimited, the input/output amount can vary, thus those two cover most cases.
```

Comment:
Baaah, the suggestion thing created duplicate lines :/

Reply:
I've fixed it.